### ISSUE TITLE
dude, where is the actual app like the image seen in the readme?

### ISSUE THREAD

**Issue by maskedband1t** [auth0-blog/redux-auth0#2]
I followed all your instructions, and I cannot get to the page seen in your readme all I can find if I add /api/jedis is some json

**Comment by chenkie** [comment]
Are you sure you're serving both the backend and the front end?

**Comment by maskedband1t** [comment]
can you explain?

**Comment by chenkie** [comment]
It sounds like you may only have the server running :) Are you sure that you've also started the front end as well? These are the steps (from the readme):

``` bash
cd server
npm install
node server.js

cd ..
npm install
npm start
```

**Comment by maskedband1t** [comment]
Yeah I followed the steps but I did not get anything like the image
